util/devel/chplspell-dictionary.fileids.json

```diff
@@ -596,6 +599,9 @@
   "eda6a798-12c0-11ed-a8be-cd81bce0efc4": [
     "compiler/dyno/lib/util/terminal.cpp"
   ],
+  "3eb6fb72-300f-11ed-b71b-d7492178b4dc": [
+    "compiler/dyno/tools/chpldoc/arg.cpp"
```

Comment:
Is it worth removing entries for files that have been removed?  I suspect there's some files related to the old parser implementation still listed in here that have now been removed (and eventually we'll remove the original chpldoc files, which is what prompted me to think of it)

Reply:
Good call!  There were rather a lot.